client/state/themes/actions.js

```diff
@@ -834,6 +836,20 @@ export function hideThemePreview() {
 	};
 }
 
+export function receiveThemeFilters( data ) {
+	return {
+		type: THEME_FILTERS_REQUEST_SUCCESS,
+		data,
+	};
+}
+
+export function receiveThemeFiltersFailure( error ) {
+	return {
+		type: THEME_FILTERS_REQUEST_FAILURE,
+		error,
+	};
+}
```

Comment:
@seear notice how these actions are essentially empty. I'm guessing that the reducers already exist to handle them? My claim with the data layer is that we have no use/need for `_SUCCESS` and `_FAILURE` action types. Instead of these we can dispatch semantically-meaningful actions in response to the API requests such as `ADD_THEME` or `NOTICE_CREATE`